src/api/app/views/webui/package/_rpmlint_log.html.haml

```diff
@@ -1,7 +1,7 @@
 - @log.lines.each do |line|
-  - if line =~ /\w+(?:\.\w+)+: W: /
+  - if line.match?(/\w+(?:\.\w+)+: W: /)
```

Comment:
well, shouldn't this be in a helper?

Reply:
@coolo yes, but the PR is not introducing the problem but fixing a problem with it. You can send a  PR :wink: